rskj-core/src/main/resources/reference.conf

```diff
@@ -354,6 +354,8 @@ rpc {
              enabled: "true"
          }
     ]
+
```

Comment:
Also here I would remove this space here to preserve the style.

Reply:
done